scripts/renderers.py

```diff
@@ -99,14 +159,81 @@ def sorting_header(key):
         output.extend(header)
         return output
 
-    def render_csv_report(self, envs_ida_toggle_data, file_path="report.csv", toggle_types=None, header=None):
+    def combine_envs_data_under_toggle_identifier(self, envs_data, type_filter=None):
         """
-        takes data, processes it, and outputs it in csv form
+        envs data is structured: envs->ida->toggles, this converts to (toggle, ida, toggle_type)->{env1_toggle_data, env2_toggle_data}
         """
-        toggles_data = self.transform_toggle_data_for_csv(envs_ida_toggle_data)
-        data_to_render = self.filter_and_sort_toggles(toggles_data, toggle_types)
-        header = self.get_sorted_headers_from_toggles(data_to_render, header)
-        self.write_csv(file_path, data_to_render, header)
+        toggles_data = {}
```

Comment:
**NOTE TO SELF:** This is where I left off on the review.  Submitting to give some early feedback.